integration-tests/DashboardPage.js

```diff
@@ -83,10 +84,12 @@ export class DashboardPage {
         await this.showsConnectionScreen()
         await this.screenshot(name + '-state-connection.png', opts)
         await this.goBack()
-        await this.viewTrackerCompanies()
-        await this.showsTrackersScreen()
-        await this.screenshot(name + '-state-trackers.png', opts)
-        await this.goBack()
+        if (this.shouldScreenshotTrackersScreen(state)) {
```

Comment:
I would just look if the link exists first:

```suggestion
        if (await this.shouldScreenshotTrackersScreen()) {
```